src/asset_verifier.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity >=0.4.24;
+
+contract IAssetManager {
+  function isAssetValid(bytes32 asset) external view returns (bool);
+}
+
+contract AssetVerifier {
```

Comment:
Let's inline this to NFT. KISS.

Reply:
This was inlined last time I pushed them. I think @mikiquantum it into a separate one.